packages/core/src/classes.ts

```diff
@@ -440,6 +440,8 @@ export interface Permission extends Doc {
 export interface Account extends Doc {
   email: string
   role: AccountRole
+
+  person?: Ref<Doc>
```

Comment:
In this case we can throw away `PersonAccount`

Reply:
Yep, on next step.